src/test/java/com/synopsys/integration/alert/performance/utility/AuditCompleteWaitJobTask.java

```diff
@@ -96,13 +109,12 @@ private Set<String> getJobIdsFromAuditEntries() throws IntegrationException {
                 .collect(Collectors.toList());
 
             intLogger.debug(String.format("Performance: Found %s audit entries discovered. ", auditEntryPageModel.getContent().size()));
-            // TODO add this check when we want to validate sending to the channel as well.
-            //            boolean anyPending = jobAuditModels.stream()
-            //                .anyMatch(Predicate.not(this::jobFinished));
-            //            if (anyPending) {
-            //                intLogger.info("Performance: Some audit entries are still processing. Continuing...");
-            //                return Set.of();
-            //            }
+            boolean anyPending = jobAuditModels.stream()
+                .anyMatch(Predicate.not(this::jobFinished));
+            if (anyPending) {
+                intLogger.info("Performance: Some audit entries are still processing. Continuing...");
```

Comment:
Is it worth listing out the processing audit entries here? Something like:
```
if (anyPending) {
                intLogger.info("Performance: Some audit entries are still processing. Continuing...");
                forEach (pendingJobAuditModels) {
                            intLogger.info("Processing Job: %s", jobThatsProcessing);
                }
}
```

I'm not sure how granular we want to be - feel free to disregard!